packages/@tinacms/cli/src/next/vite/index.ts

```diff
@@ -0,0 +1,113 @@
+import path from 'path'
+import { InlineConfig, Plugin, splitVendorChunkPlugin } from 'vite'
+import { Database } from '@tinacms/graphql'
+import { tinaTailwind } from './tailwind'
+import { ConfigManager } from '../config-manager'
+import normalizePath from 'normalize-path'
+
+export const createConfig = async (
```

Comment:
Might be nitpicky, but can the `createConfig` function be take a single object argument instead of a list of 6 args.


I find
```ts
 const config = await createConfig(
    configManager,
    database,
    apiURL,
    [],
    noSDK,
    true
  )
```

Hard to understand.